components/base/kinematicbase/differentialDrive.go

```diff
@@ -82,6 +87,7 @@ type differentialDriveKinematics struct {
 	logger                        logging.Logger
 	planningFrame, executionFrame referenceframe.Model
 	options                       Options
+	noLocalizerCacheInputs        []referenceframe.Input
```

Comment:
[note] Added the ability to return an estimated `CurrentInputs()` for non-localizer differential drives by assuming move actions complete exactly, and resetting once `GoToInput` call is complete.